.gitmodules

```diff
@@ -10,3 +10,6 @@
 	path = imported/aste
 	url = https://github.com/precice/aste.git
 	branch = develop
+[submodule "imported/micro-manager"]
+	path = imported/micro-manager
+	url = https://github.com/precice/micro-manager.git
```

Comment:
Why no `branch` for the `micro-manager` here?
I guess the workflow is failing because the pages don't yet exist in the default branch (main): https://github.com/precice/micro-manager/tree/main/docs

```
       Subproject: imported/openfoam-adapter/docs
      Adding pages: README.md, openfoam-support.md, config.md, get.md, extend.md
         Tutorials: No subproject found at imported/micro-manager/docs
        Subproject: imported/micro-manager/docs
                    ------------------------------------------------
      Jekyll 4.2.2   Please append `--trace` to the `build` command 
                     for any additional information or backtrace. 
                    ------------------------------------------------
/home/runner/work/precice.github.io/precice.github.io/website/_plugins/subprojects.rb:22:in `block in generate': No subproject found at imported/micro-manager/docs (RuntimeError)
```

Reply:
I don't know, but I am confused by this line:

```
Tutorials: No subproject found at imported/micro-manager/docs
```

Where should the `Tutorials:` come from?